TransitionsApp/SecondViewController.swift

```diff
@@ -7,36 +7,71 @@
 
 import Foundation
 import UIKit
+enum Colour{
+    case blue
+    case red
+    case green
+    
+    var colorSetting: UIColor {
+        switch self {
+        case .blue:
+            return.blue
+        case .red:
+            return.red
+        case .green:
+            return.green
+        }
+    }
+}
 
 final class SecondViewController: UIViewController {
-    @IBOutlet private weak var colorButton: UIButton!
-    @IBOutlet private weak var closeButton: UIButton!
+    @IBOutlet private weak var redButton: UIButton!
+    @IBOutlet weak var greenButton: UIButton!
+    @IBOutlet weak var cioseedButton: UIButton!
```

Comment:
これはタイポ(誤字)かな！
細かいけど修正しよう！

```suggestion
    @IBOutlet weak var closedButton: UIButton!

```